### PR TITLE
Bug/GAT-694 validation on spatial filter input

### PR DESCRIPTION
**Pull request by kandaj**

### CODE REVIEW

**Review by cdjreekie — APPROVED** [review]
lgtm

File: src/pages/DatasetOnboarding/components/TypeaheadAsyncCustom/TypeaheadAsyncCustom.jsx [hunk 1]
```diff
@@ -48,31 +52,44 @@ function TypeaheadAsyncCustom(props) {
         }
     };
 
+    const errorHandler = value => {
+        value ? (isNaN(value) ? setShowError(false) : setShowError(true)) : setShowError(false);
```
Comment (ThinkingMonkeyLtd) [inline src/pages/DatasetOnboarding/components/TypeaheadAsyncCustom/TypeaheadAsyncCustom.jsx:56]: Isn't this the same as:    setShowError(!isNaN(value))

File: src/pages/DatasetOnboarding/components/TypeaheadAsyncCustom/TypeaheadAsyncCustom.jsx [hunk 2]
```diff
@@ -48,31 +52,44 @@ function TypeaheadAsyncCustom(props) {
...
+        setIsLoading(false);
+    };
+
+    const handleInputChange = async value => {
```
Comment (ThinkingMonkeyLtd) [inline src/pages/DatasetOnboarding/components/TypeaheadAsyncCustom/TypeaheadAsyncCustom.jsx:60]: Why async?

Comment (kandaj) [reply src/pages/DatasetOnboarding/components/TypeaheadAsyncCustom/TypeaheadAsyncCustom.jsx:60]: copied the code and forgot to remove :-)

File: src/pages/DatasetOnboarding/components/TypeaheadAsyncCustom/TypeaheadAsyncCustom.jsx (hunk 1 above)
Comment (kandaj) [reply src/pages/DatasetOnboarding/components/TypeaheadAsyncCustom/TypeaheadAsyncCustom.jsx:56]: oops. i'm a bad programmer.

**Review by cdjreekie — APPROVED** [review]
lgtm

**Pull request merged**